crates/telio-nurse/src/aggregator.rs

```diff
@@ -0,0 +1,689 @@
+use std::{collections::HashSet, time::Duration};
+
+use tokio::time::Instant;
+
+use parking_lot::RwLock;
+use serde::{ser::SerializeTuple, Serialize, Serializer};
```

Comment:
Serialization is not used or tested - is it really needed? If it will be needed in future, please add some tests for it. Especially for the custom implementation.

Reply:
Riight, I think I will remove it for now